src/elfpy/markets.py

```diff
@@ -299,16 +299,16 @@ def close_short(
             d_token_asset=trade_result.market_result.d_bonds,
             d_bond_buffer=-trade_amount,
         )
+        d_worst_case_scenario = trade_amount  # in the worst case for the short, p=1 and they owe the face value
```

Comment:
Please just use the `trade_amount` variable instead of introducing this variable, and use comments & docstrings to explain how/why/when it is the worst_case_scenario.